app_check/integration_test/src/integration_test.cc

```diff
@@ -956,226 +795,4 @@ TEST_F(FirebaseAppCheckTest, TestFunctionsSuccess) {
   }
 }
 
-TEST_F(FirebaseAppCheckTest, TestFunctionsFailure) {
```

Comment:
Why is this test removed?

Reply:
All the tests that involved running without App Check defined were seeing problems on mobile.  Once the Factory and Provider have been gotten for an App, it remembers it forever, so these would always fail.  On desktop, it was acting flakey, I'm not sure why since I never had problems with it locally, but would need more time to investigate it.